src/main/java/com/deusgmbh/xcusatio/ui/editor/excuse/ExcuseEntryListPane.java

```diff
@@ -0,0 +1,89 @@
+package com.deusgmbh.xcusatio.ui.editor.excuse;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+
+import com.deusgmbh.xcusatio.data.excuses.Excuse;
+import com.deusgmbh.xcusatio.ui.editor.EntryListPane;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.cell.PropertyValueFactory;
+
+public class ExcuseEntryListPane extends EntryListPane {
+    private static final String REQ_COLUMN_TEXT_VARNAME = "text";
+    private static final String REQ_COLUMN_TAGS_VARNAME = "tags";
+    private static final String REQ_COLUMN_TEXT_TITLE = "Ausrede";
+    private static final String REQ_COLUMN_TAGS_TITLE = "Tags";
+    protected static final String NEW_EXCUSE_DEFAULT_TEXT = "Bitte hier neue Ausrede eingeben";
+
+    // @Tobias: Is there a possibility to bring the table view as protected
+    // TableView to the parent class?
+    // (Generic Type which is first declared with the constructor.)
```

Comment:
Das könnte man mit einer abstrakten Klasse realisieren. Das ist ne Mischung aus einer normalen Klasse und einem Interface.
Aber was möchtest du damit bezwecken?

Reply:
nur das 2 methoden die fast gleich sind im parent stehen und nicht zweimal in den entsprechenden childs. Aber denke das ist mit ner abstrakten Klasse dann zu viel Aufwand für den wenigen Mehrzweck...

Reply:
done